provisioning/miniprov/hare_mp/main.py

```diff
@@ -80,6 +83,10 @@ def create_logger_directory(log_dir):
 
 def setup_logging(url) -> None:
     provider = ConfStoreProvider(url)
+    filepath = '/etc/machine-id'
```

Comment:
let's move this code into a function, `check_machine_id()` and invoke it from main before setup_logging().

Reply:
done